examples/account-management.ipynb

```diff
@@ -0,0 +1,264 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from qiskit_ibm_runtime import IBMRuntimeService"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Examples\n",
+    "\n",
+    "## Account Management\n",
+    "### Load account"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# load default account (heuristic to be determined)\n",
+    "account1 = IBMRuntimeService.load_default_account()"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# load named account from default JSON configuration file\n",
+    "account2 = IBMRuntimeService.load_account(name=\"my-account\")"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# load account configured via environment variables\n",
+    "account3 = IBMRuntimeService.load_account_from_environment()"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Save account\n"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# create and save cloud account\n",
+    "IBMRuntimeService.save_account(\n",
+    "    name=\"cloud-account-1\",\n",
+    "    account={\"token\": \"123\", \"locator\": \"Quantum Service-34\"},\n",
+    ")"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# create and save legacy account\n",
+    "IBMRuntimeService.save_account(\n",
+    "    name=\"legacy-account-1\",\n",
+    "    account=dict(type=\"legacy\", token=\"123\"),\n",
+    ")"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# overwrite legacy account and add default HGP information\n",
+    "IBMRuntimeService.save_account(\n",
+    "    name=\"legacy-account-1\",\n",
+    "    account=dict(type=\"legacy\", token=\"123\", hub=\"ibm-q\", group=\"open\", project=\"main\"),\n",
```

Comment:
hub, group and project should be direct parameters to this method.

Reply:
I can surely change that but would like to understand the benefits of passing it as direct parameters first. 
 
The intention for using the proposed account parameter is to
- ease duck typing of input and output parameters
- make it obvious which parameters belong to the saved account, and which control how to save it.

https://github.com/Qiskit/qiskit-ibm-runtime/blob/39ad0af300318bb26bd8ecfe1ffea8dbaad02b79/qiskit_ibm_runtime/accounts/management.py#L42-L49

Reply:
We typically get pushed back on nested structures. It's the understanding that if it's flat it's easier to use and document. I know it's totally wrong to have too many parameters in a function as per Martin Fowler, but that's how people seem to like it for this library.